src/hardware/driver/adc_internal.rs

```diff
@@ -31,6 +31,11 @@ pub struct AdcInternal {
 }
 
 impl AdcInternal {
+    // ADC sampling time (aka time to charge the sampling capacitor).
+    // For the RC circuit on the Stabilizer CPU ADC hearder, sampling times of more than
+    // 160 ns were not observed to have an effect on the sampled values.
```

Comment:
Ok. But it's always strictly better to derive things (like the minimum sampling time) instead of eyeballing them.